var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -119,7 +119,26 @@ class Cp2k(MakefilePackage, CMakePackage, CudaPackage, ROCmPackage):
     variant("dftd4", when="@2024.2:", default=False, description="Enable DFT-D4 support")
     variant("mpi_f08", default=False, description="Use MPI F08 module")
     variant("smeagol", default=False, description="Enable libsmeagol support", when="@2025.2:")
-
+    variant(
+        "pw_gpu",
+        default=True,
+        description="Enable FFT calculations on GPU (off by default for now)",
+        when="@2025.2: +cuda",
```

Comment:
```suggestion
    variant(
        "pw_gpu",
        default=True,
        description="Enable FFT calculations on GPU",
        when="@2025.2: +cuda",
```
This is `default=True`? I'd leave this comment out entirely, since `spack info` shows the defaults and there is no risk with the description going out of sync with the default value.